### DISCUSSION TITLE
Can't profile my simple program.

### DISCUSSION THREAD

**Discussion by weisuofayuba** [google/orbit#4650] [Q&A]
I write a test program and profile it with Orbit,
`int func1(void)
{
    int sum=0;
    for(int i=0;i<100000;i++)
    {
        sum+= i;
    }
    return sum;
}
int func2(void)
{
    int sum=0;
    for(int i=0;i<1000000;i++)
    {
        sum+= i;
    }
    return sum;
}
int main(int argc, char **argv)
{
    while(1){
        func1();
        func2();
    }
    
    return 0;
}`
But Orbit doesn't show any info in "Top-Down" and "Bottom-Up".

**Comment by florian-kuebler** [comment]
>But Orbit doesn't show any info in "Top-Down" and "Bottom-Up".

What do you mean by that? Can you provide a screenshot?

A couple of questions to be answered:
* How is the program compiled? Are symbols included in the binary or is there a separate `.debug` file (or similar). You might want to check out the [documentation](https://github.com/google/orbit/blob/main/documentation/DOCUMENTATION.md#additional-symbol-locations) on how to setup symbols.
* What operating system are you running your target application on?
* What version of Orbit are you running?
* How have you started `OrbitService`? Is it running as root?
